packages/opentelemetry-node/test/OTEL_SDK_METRICS.test.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const {test} = require('tape');
+const {runTestFixtures} = require('./testutils');
+
+/** @type {import('./testutils').TestFixture[]} */
+const testFixtures = [
+    {
+        name: 'basic scenario with no sampling configutarion involved (all spans sampled)',
+        args: ['./fixtures/use-http-server-metrics.js'],
+        cwd: __dirname,
+        env: {
+            NODE_OPTIONS: '--import=@elastic/opentelemetry-node',
+            OTEL_METRICS_EXPORTER: 'otlp,console',
```

Comment:
Was the `console` inclusion for debugging? SHould it be removed? Could drop the whole envvar and rely on the default `otlp`, perhaps.